user/inc/Arduino.h

```diff
@@ -4,6 +4,7 @@
 
 #define PARTICLE_ARDUINO_COMPATIBILITY
 #include "Particle.h"
+#include "spark_wiring_arduino.h"
```

Comment:
I want to make sure your removal of this change still keeps the intent that:
```
#include "application.h"
#include "Arduino.h"
// compatibility macros should be defined here
```

It wasn't the case due to `Arduino.h` loading `Particle.h` loading `application.h` which had already been loaded without compatibility macros and wasn't loaded a second time due to the guard clauses.